src/components/Forecast/index.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import "./style.css";
+
+const Forecast = (props) => {
+    const { weather } = props;
+
+    const days = {
+        Mon: "Monday",
+        Tue : "Tuesday",
+        Wen : "Wednesday",
+        Thu : "Thursday",
+        Fri : "Friday",
+        Sat : "Saturday",
+        Sun : "Sunday"
+      };    
+      
+      const months = {
+        Jan : "Januar",
+        Feb : "Februar",
+        Mar : "March",
+        Apr : "April",
+        May : "May",
+        Jun : "June",
+        Jul : "July",
+        Aug : "August",
+        Sep : "September",
+        Oct : "October",
+        Nov : "November",
+        Dec : "December",
+      };
+
+      const daysMapper = (shortage) => {
+        return days[shortage];
+      }
+    
+      const monthsMapper = (shortage) => {
+        return months[shortage];
+      }
+
+      console.log(weather);
+      console.log(new Date(weather.dt).toString());
+
+      const day = new Date(weather.dt).toString().substring(0,3).trim();
+      const month = new Date(weather.dt).toString().substring(7,3).trim();
```

Comment:
Podívej se na metody Date - třebe getDay, getMonth ;) takhle to je hodně náchylný na změnu, počítat to dle pozice čísel ve stringu.